packages/apps/crm/src/screens/opportunity/OpportunityListScreen.js

```diff
@@ -140,14 +130,11 @@ const OpportunityListScreen = ({navigation}) => {
               rightTitle={I18n.t('Crm_AssignedToMe')}
               onSwitch={() => setAssigned(!assigned)}
             />
-            <AutoCompleteSearch
-              objectList={opportunityList}
-              value={opportunity}
-              onChangeValue={setOpportunity}
-              fetchData={fetchOpportunityFilter}
-              placeholder={I18n.t('Crm_Opportunity')}
+            <OpportunitySearchBar
+              onChange={() => {}}
+              defaultValue={''}
```

Comment:
This should be transformed by default values in the component, leaving this is here is dirty